### PR TITLE
Ruthless trimming

### PR DESCRIPTION
**Pull request by kcranston**
This PR cuts two pages and ~1000 words. New stats are 16 pages and 7252 words (from a starting point of 18 pages and 8204 words). Each commit covers only one section, so we can cherry-pick specific commits if any objects to any of the changes. 

Major changes:
- removed a good chunk of the lengthy description of various software file types that might go into `src` (analysis files vs controller scripts)
- renamed 'Version Control' to 'Keeping Track of Changes' because the text seemed unclear when we used the phrase 'version control' whether we were talking generally or specifically about VCS
- It wasn't clear whether the 'What Not to Put Under Version Control' section was meant to apply to manual and VCS workflows, or just the latter. Most of the text was specific for VCS, so I re-wrote it for that specifically
- Also in the 'What Not to Put Under Version Control', I reduced the number of points to concerns about 1) file / repo size and 2) file types not amenable to tracking changes. 
- some replacement of passive voice with active voice in places where that shortened the text significantly. I resisted the urge to do this throughout. 
- removed the box about 'Linking vs Embedding' because it seemed like too much emphasis on a relatively minor point

### CODE REVIEW

**Comment by beroe** [comment]
`++++` These are great edits, Karen!

There are a couple split infinitives, which I normally don't care about, but maybe `to precisely refer` could be changed to `to refer precisely`?

**Comment by jennybc** [comment]
Thanks @kcranston! I'm about to run teach (Data Carpentry!) but my plan is to do a pass during travels that start tomorrow. So it would be great if we could merge this so I can pull and work from that.

File: good-enough-practices-for-scientific-computing.tex [hunk 1]
```diff
@@ -252,29 +244,16 @@ \section{Data Management}\label{sec:data}
   also makes it much easier to link data sets together.
 
 \item
-  \recommend{Include complementary data in files that meet the same
-    standards}.  Raw data frequently does not fully explain
+  \recommend{Include complementry data in additional files}.  Raw data
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:247]: complementary (but I'll fix after merge during spell-check)

File: good-enough-practices-for-scientific-computing.tex [hunk 2]
```diff
@@ -517,23 +486,19 @@ \section{Collaboration}\label{sec:collaboration}
...
+    software into other projects. 
   \end{framed}
 
-\item    
+\item
   \recommend{Make the project citable} by including a
   \texttt{CITATION} file in the project's home directory that
-  describes how to cite this project as a whole, and where to find,
-  and how to cite, any data sets, code, figures, and other artifacts
+  describes how to cite this project as a whole, and where to find / how to cite
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:500]: or parenthesize? (I'll take a look after merging)

File: good-enough-practices-for-scientific-computing.tex [hunk 3]
```diff
@@ -710,6 +654,7 @@ \section{Project Organization}\label{sec:project}
 .
 |-- CITATION
 |-- README
+|-- LICENSE
```
Comment (gvwilson) [inline good-enough-practices-for-scientific-computing.tex:657]: Right :-)

**Pull request merged**